test/features/app.test.js

```diff
@@ -23,7 +23,7 @@ describe('Guess Vu', () => {
 
   describe('Name form', () => {
     test('redirects to chatroom after signup', async () => {
-      await page.waitForSelector('.Form');
+      await page.waitForSelector('#Form');
```

Comment:
Sure this works? Let's leave it for now though, all of the tests need to be fixed to adjust for imported styles. Let's make a ticket for this.